extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -26,12 +26,14 @@ package stackdriver.config.v1alpha1;
 
 import "google/protobuf/duration.proto";
 
-// next id: 9
-
+// next id: 10
 message PluginConfig {
   // Optional. Controls whether to export server access log.
   bool disable_server_access_logging = 1;
 
+  // Optional. Controls whether to export TCP server access log.
+  bool disable_tcp_server_access_logging = 9;
```

Comment:
Shall we just use `disable_server_access_logging` to control both tcp and http logging?

Reply:
Good idea.. removing